pithos/plugins/mpris.py

```diff
@@ -424,6 +420,20 @@ def _get_metadata(self, window, song):
 
         return metadata
 
+    def _metadata_equal(self, m1, m2):
```

Comment:
Sorry I meant `.equal()` rather than `.compare()` I think it can work on more types.